src/types/ajv/BroadcastFinalStateReq.ts

```diff
@@ -0,0 +1,31 @@
+import { addSchema, addSchemaDependency } from '../../utils/serialization/SchemaHelpers'
+import { AJVSchemaEnum } from '../enum/AJVSchemaEnum'
+
+const BroadcastFinalStateReqSchema = {
+    type: 'object',
+    properties: {
+        txid: { type: 'string' },
+        stateList: {
+        type: 'array',
+        items: { $ref: AJVSchemaEnum.WrappedDataResponse },
+        },
+    },
+    required: ['txid', 'stateList'],
+}
+
+export function initBroadcastFinalStateReq(): void {
```

Comment:
This needs to be added to Helpers.ts for initiation. The Ajv schema validation will not be proper without this.